lib/run.js

```diff
@@ -87,6 +99,13 @@ function run (opts, cb) {
   url.setupClient = opts.setupClient
   url.timeout = opts.timeout
   url.requests = opts.requests
+  url.responseMax = opts.maxConnectionRequests
+
+  if (!url.responseMax) {
+    url.responseMax = typeof opts.maxOverallRequests === 'number'
+                      ? (opts.maxOverallRequests / opts.connections) : false
+    // this needs work if ((maxOverallRequests % opts.connections) > 0)
+  }
```

Comment:
I expect you'll see this. If the response max is set with the MaxConnectionReqs this will be jumped over, as MaxConnectionReqs takes precedence. If it is not set, we check if the maxOverallReqs is a number to avoid a NAN division error. I originally had:
```
url.responseMax = opts.maxConnectionRequests
                      || typeof opts.maxOverallRequests === 'number'
                          ? (opts.maxOverallRequests / opts.connections)
                          : false
```
but encountered an error were url.responseMax was set to 0, when opts.maxConnectionRequests was passed in.